File: core/graph/reasoning/graph_memory_engine.py
from __future__ import annotations

from core.graph.graph_reconstruction_engine import bound_graph_memory


def graph_memory_bound(graph: dict, max_nodes: int = 5000, max_edges: int = 20000):
    return bound_graph_memory(graph, max_nodes=max_nodes, max_edges=max_edges)

File: src/graph/reasoning/graphMemoryEngine.ts
/**
 * Converted from Python: core/graph/reasoning/graph_memory_engine.py
 * @generated — WebWeaveX python→javascript library port
 */

import { boundGraphMemory } from "../graphReconstructionEngine.js";

export function graphMemoryBound(graph: any, max_nodes: any = 5000, max_edges: any = 20000): any {
  return boundGraphMemory(graph, max_nodes, max_edges);
}
export { boundGraphMemory };
